packages/components/scss/BaseStyleSheet.scss

```diff
@@ -146,7 +146,7 @@ button:focus {
 }
 
 span.btn-disabled-wrapper {
-  display: contents;
+  display: inline-block;
```

Comment:
I'm keeping this because it seems `display: contents` is making the tooltips not show up at all when disabled. There's been some changes to the button component around themeing which seems to have fixed the disabled icon buttons with tooltips when I leave this as `inline-block`